cmd/kubeadm/app/apis/kubeadm/validation/validation.go

```diff
@@ -440,7 +440,7 @@ func ValidatePodSubnetNodeMask(subnetStr string, c *kubeadm.ClusterConfiguration
 		// the pod subnet mask needs to allow one or multiple node-masks
 		// i.e. if it has a /24 the node mask must be between 24 and 32 for ipv4
 		if maskSize > nodeMask {
-			allErrs = append(allErrs, field.Invalid(fldPath, podSubnet.String(), "pod subnet size is smaller than the node subnet size"))
+			allErrs = append(allErrs, field.Invalid(fldPath, podSubnet.String(),  fmt.Sprintf("pod subnet size %s is smaller than the node subnet size %d", maskSize, nodeMask)))
 		} else if (nodeMask - maskSize) > constants.PodSubnetNodeMaskMaxDiff {
 			allErrs = append(allErrs, field.Invalid(fldPath, podSubnet.String(), fmt.Sprintf("pod subnet mask and node-mask difference can not be greater than %d", constants.PodSubnetNodeMaskMaxDiff)))
```

Comment:
since we are doing this, we can fix this one too

Reply:
Walk through the file and find another one. Fixed them all now.